### DISCUSSION TITLE
block all popups window

### DISCUSSION THREAD

**Discussion by Pantyhose-X** [uBlockOrigin/uBlock-issues#2424] [Q&A] [answered]
rules
`no-popups: * true`

Unable to block these types of new window
https://www.webroot.com/services/popuptester1.htm
https://www.w3schools.com/jsref/obj_window.asp 

https://serialeonlinesubtitrate.ro/log-in/  Click on the logo to bring up the New Tab
https://watchseinfeld.com/  Click on the logo to bring up the New Tab
What's the best way to block popups from all websites?

**Comment by uBlock-user** [comment]
That rule will only close popups, not block them.

**Reply by Pantyhose-X** [reply]
Is it possible to close all pop-ups with `window.open`?

**Reply by uBlock-user** [reply]
> Is it possible to close all pop-ups with `window.open`?

<code>domain.tld##+js(nowoif)</code>

**Comment by Pantyhose-X** [comment, marked as answer]
Thanks Close all popup window
In My rules Add `no-popups: * true`
In My filters Add 
`*##+js(set, window.open, undefined)` and 
`*##+js(abort-on-property-read.js, open)` and 
 [`*##+js(window-close-if, /plan/select?popup=noads)`](https://github.com/uBlockOrigin/uBlock-issues/wiki/Resources-Library#window-close-ifjs-)
### other ways
Disable JavaScript
